### ISSUE TITLE
Build issue with 8.1.1RC1

### ISSUE THREAD

**Issue by remicollet** [krakjoe/uopz#159]
Looks like something strange happen between 8.1.0 and 8.1.1RC1

Found in Fedora CI
https://koschei.fedoraproject.org/package/php-pecl-uopz?collection=f36

Build:
```
In file included from /usr/include/php-zts/php/Zend/zend.h:28,
                 from /usr/include/php-zts/php/main/php.h:31,
                 from /builddir/build/BUILD/php-pecl-uopz-7.1.1/ZTS/src/function.c:22:
/builddir/build/BUILD/php-pecl-uopz-7.1.1/ZTS/src/function.c: In function 'uopz_copy_function':
/usr/include/php-zts/php/Zend/zend_map_ptr.h:42:35: warning: assignment to 'HashTable **' {aka 'struct _zend_array **'} from incompatible pointer type 'HashTable *' {aka 'struct _zend_array *'} [-Wincompatible-pointer-types]
   42 |                 ZEND_MAP_PTR(ptr) = (val); \
      |                                   ^
/builddir/build/BUILD/php-pecl-uopz-7.1.1/ZTS/src/function.c:58:17: note: in expansion of macro 'ZEND_MAP_PTR_INIT'
   58 |                 ZEND_MAP_PTR_INIT(
      |                 ^~~~~~~~~~~~~~~~~

```

And test suite
```
=====================================================================
FAILED TEST SUMMARY
---------------------------------------------------------------------
uopz_add_function [tests/012.phpt]
=====================================================================

```

**Comment by remicollet** [comment]
Looks like reverting d43d4cd102648d948541aa4321f11177455146bc fixes this issue... :s

**Comment by remicollet** [comment]
Fixed  by 80df57f65243950f06435f432415aaf5f03a9d56

@krakjoe  please review.

**Comment by krakjoe** [comment]
Ta @remicollet
